src/vscode-atopile/src/ui/ui.ts

```diff
@@ -0,0 +1,256 @@
+import * as vscode from 'vscode';
+import { window, Uri } from 'vscode';
+import * as yaml from 'js-yaml';
+import * as cp from 'child_process';
+
+let statusbarAtoBuild: vscode.StatusBarItem;
+let statusbarAtoCreate: vscode.StatusBarItem;
+let statusbarAtoInstall: vscode.StatusBarItem;
+let statusbarAtoInstallPackage: vscode.StatusBarItem;
+let statusbarAtoBuildTarget: vscode.StatusBarItem;
+let statusbarAtoLaunchKiCAD: vscode.StatusBarItem;
+
+let builds: string[] = [];
+interface AtoYaml {
+    atoVersion: string;
+    builds: {
+        [key: string]: {
+            entry: string;
+        };
+    };
+    dependencies: string[];
+}
+const atopileInterpreterSetting = 'atopile.interpreter';
```

Comment:
I believe there's a `resolveInterpreter` function. This will use the python interpreter unless an atopile-specific one is set, which should be a bit better.
Using the other setting also activates the environment when starting new terminals by default and a few other good things.

Reply:
The commands I tested do with with the `atopile.interpreter` setting set though 🥳

Reply:
I looked into `resolveInterpreter`. I couldn't find much documentation for it, but from what I could figure out, if you pass it a path to a python interpreter, it will provide you with version, path, compile info etc.

Reply:
No stress, we can check it out later: https://github.com/atopile/atopile/issues/231